packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -41,6 +41,10 @@ export const messages = {
 
   force_auth_web_login_authorize_dev_hub_text: 'SFDX: Authorize a Dev Hub',
 
+  force_apex_class_create_enter_file_name: 'Enter desired filename',
+  force_apex_class_create_enter_dir_name:
+    "Enter desired directory (Press 'Enter' to confirm or 'Escape' to cancel)",
```

Comment:
QuickPick input window does not indicate how to confirm or cancel so included it in the placeholder text.